app/javascript/packages/document-capture/components/file-input.jsx

```diff
@@ -194,11 +194,12 @@ function FileInput(props, ref) {
   }
 
   /**
+   * @param {string} fileLabel String velue of the label for input to display
    * @param {Blob|string|null|undefined} fileValue File or string for which to generate label.
    */
-  function getLabelFromValue(fileValue) {
+  function getLabelFromValue(fileLabel, fileValue) {
     if (fileValue instanceof window.File) {
-      return fileValue.name;
+      return `${fileLabel} - ${fileValue.name}`;
```

Comment:
Should we include the label in all of the possible return paths of this function, not just the one where we know the file's name?

Reply:
Yes I added a label to the second return statement. The last return path doesn't need it. I tested with a screen reader and it looks like the label field will take precedence over that.